admin/theme-customizer.php

```diff
@@ -316,7 +316,6 @@ function responsive_customize_register( $wp_customize ) {
 			'burf_setting_posts_sidebar_bottom',
 			array(
 				'type'    => 'option',
-				'default' => true,
```

Comment:
This line was the culprit for why Customizer > Content Options > "Keep the posts sidebar on bottom" was not persisting when unchecked, saved. The page would always keep the box checked because we'd remove the option, publish changes, it wouldn't exist on page load therefore would default to `true` due to this line.

Removing this line causes the box to remain unchecked.